sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -451,6 +452,19 @@ private[spark] object SQLConf {
     doc = "When true, we could use `datasource`.`path` as table in SQL query"
   )
 
+  val PARSER_SUPPORT_QUOTEDID = stringConf("spark.sql.parser.supportQuotedIdentifiers",
```

Comment:
I ported this directly form Hive. It has only two options:
- ```none```: no quoting. I'll map this to false.
- ```column```: quoting. I'll map this to true.